packages/okta-angular/test/e2e/harness/README.md

```diff
@@ -21,6 +21,6 @@ export PASSWORD={password}
 ```bash
 # At project root
 cd packages/okta-angular
-npm install
-npm test
+yarn install
```

Comment:
Is this necessary? I.e. in general with workspaces, you run a `yarn install` in the root dir, and there's only one yarn.lock that sits at the root. Couple things to revisit over this entire PR:
- All the places where we're referencing `yarn installs` in the package documentation
- All the packages that have their own `yarn.lock`